src/en/administration/from_scratch.md

```diff
@@ -207,14 +217,17 @@ systemctl restart lemmy-ui
 
 ### Pict-rs
 
+If you did **not** use the `--features embed-pictrs` flag, then this script below is necessary for installing/updating Pict-rs as a standalone server.
+Otherwise, pict-rs should update with lemmy_server.
```

Comment:
This only makes sense if you mention in the beginning that pictrs can be installed as standalone.

Reply:
Yes dependencies would be the same either way.